sycl/source/detail/os_util.cpp

```diff
@@ -191,7 +191,7 @@ OSModuleHandle OSUtil::getOSModuleHandle(const void *VirtAddr) {
     return 0;
   }
   if (PhModule == GetModuleHandleA(nullptr))
-    return OSUtil::ExeModuleHandle;
+    return 0;
```

Comment:
Now caller won't know that it is the exe module handle. You probably need OSUtil::ExeModuleHandle to os-specific module handle conversion util function for 'exe' module and use it on the caller side, where os-specific API is used (AFAIU).

Reply:
Thanks, I fixed